src/app/ui/components/button/button.tsx

```diff
@@ -9,11 +9,11 @@ export type ButtonProps = Omit<
 > &
   ButtonVariantProps;
 
-export function LeatherButton(props: ButtonProps) {
-  const { children, variant, fullWidth, invert, size, type = 'button', ...rest } = props;
+export function Button(props: ButtonProps) {
```

Comment:
I think we can try and get rid of `invert` for `Button`. From what I can see it's just used on the `welcome.layout` and I will need to do some custom code there anyway for the new design where we ignore the theme.

Reply:
Tend to agree — We also don't have plans for doing this elsewhere right now, so fee free to 🔥

Reply:
Ok, np, I can remove.

Reply:
@pete-watters I unresolved this so we can establish how to handle this change b/w our changes. I looked at your draft code and it appears we will have some conflicts with removing the `invert` and the change separating out the `Link` component. Would it make sense for me to just leave what I've done bc I will likely merge first, and then you can remove the invert code in your rebase?

Reply:
Leave what you have done and I can update and remove when ready to avoid headache for you as I will need to refactor it anyway. I just wanted to raise the topic when I saw it here

Thanks! 🙇